### PR TITLE
Added Projectile Code

### PR DESCRIPTION
**Pull request by esthergassner**

### CODE REVIEW

File: src/com/company/Projectile.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    public double calcX()
+    {
+        return Math.cos(Math.toRadians(degrees)) * velocity * time;
+    }
+
+    public double calcY()
+    {
+        return Math.sin(Math.toRadians(degrees)) * velocity * time - 9.8 * .5 * Math.pow(time, 2);
```
Comment (andrewoid) [inline src/com/company/Projectile.java]: instead of doing `Math.pow(times,2)`
just do `time * times`

**Comment by esthergassner** [comment]
Aha good point

File: src/com/company/Main.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        for (int second = 0; second < 120; ++second)
+        {
+            for (Projectile projectile : projectiles)
+            {
+                projectile.addTime(1);
+            } //should this go before or after adding new projectiles?
+            int nrProjectiles = rand.nextInt(6);
+            for (int ix = 0; ix < nrProjectiles; ++ix)
+            {
+                projectiles.add(new Projectile((double)rand.nextInt(171) + 50.0, (double)rand.nextInt(181) + 30.0));
```
Comment (andrewoid) [inline src/com/company/Main.java]: These are "special numbers" you should create named constants from them.

File: src/com/company/Main.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package com.company;
```
Comment (andrewoid) [inline src/com/company/Main.java]: The package name should be `yourlastname.physics`

File: src/com/company/Projectile.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
+package com.company;
```
Comment (andrewoid) [inline src/com/company/Projectile.java]: This should be `yourlastname.physics`

**Review by andrewoid — CHANGES_REQUESTED** [review]

**Review by andrewoid — APPROVED** [review]

**Pull request merged**